todo_app/config.py

```diff
@@ -1,13 +1,5 @@
 import os
 
-
-class Config:
-    """Base configuration variables."""
-    SECRET_KEY = os.environ.get('SECRET_KEY')
-    if not SECRET_KEY:
-        raise ValueError("No SECRET_KEY set for Flask application. Did you follow the setup instructions?")
-
-
 class TrelloConfig:
     """ Configuration variables for Trello API """
     BOARD_ID = os.environ.get('BOARD_ID')
```

Comment:
This class still causes errors when I run the integration tests, due to it picking up the environment variables before they've been loaded in. We can resolve that by moving the variables to a constructor, i.e. something like:
```python
class TrelloConfig:
    def __init__(self):
        """ Configuration variables for Trello API """
        self.BOARD_ID = os.environ.get('BOARD_ID')
        ...
```

And then accessing the class anywhere it's used by creating an instance, rather than directly, e.g. with `TrelloConfig().API_KEY` rather than `TrelloConfig.API_KEY`

This same adjustment allows your end-to-end tests to work from the command line

Reply:
Thanks, Jack! This comment was very useful! I've added the fix to #6 branch